tutorial/Lab1/README.md

```diff
@@ -69,101 +74,135 @@ In this part of the lab, we will deploy the application using the Kubernetes cli
 
        ![Guestbook](../images/guestbook-page.png)
 
-# Deploy the application using Helm
+## Scenario 2: Deploy the application using Helm
 
 In this part of the lab, we will deploy the application by using Helm. We will set a release name of `guestbook-demo` to distinguish it from the previous deployment. The Helm chart is available [here](../../charts/guestbook). Clone the [Helm 101](https://github.com/IBM/helm101) repo to get the files:
-```$ git clone https://github.com/IBM/helm101``` .
+
+```console
+git clone https://github.com/IBM/helm101
+```
```

Comment:
Moved to right folder before Helm command. I added

```
cd helm101/charts
````

Reply:
The cd command is here:

https://github.com/IBMAppModernization/helm101/blame/231b11618c1c79cb6b7e4f68165385f9b0c16842/tutorial/Lab1/README.md#L115